### ISSUE TITLE
Publish a new version over the "prototype"

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-dc#46]
At Thursday's meeting, it was suggested that we could publish the current CCK on top of the existing "prototype" to get more testing in the wild.  This would only be done after a good round of QA.

**Comment by samreid** [comment]
On hold until #47 is complete.

**Comment by samreid** [comment]
@kathy-phet we discussed publishing another "prototype" version once dev testing is complete: https://github.com/phetsims/circuit-construction-kit-dc/issues/50#issue-235580092

Dev testing is complete, and raised 15 issues, several of which are fixed or non-severe: https://github.com/phetsims/QA/issues/33

@kathy-phet how would you like to proceed with a second "prototype" version?

**Comment by samreid** [comment]
Here's a dev version from after dev testing and before code review (in case upcoming code review destabilizes things):
http://www.colorado.edu/physics/phet/dev/html/circuit-construction-kit-dc/1.0.0-dev.154/circuit-construction-kit-dc_en.html

**Comment by samreid** [comment]
@kathy-phet said if it is only an hour or so to publish over the prototype, then go for it.  Then let @oliver-phet and @kathy-phet know and they can decide if they want to email/etc.

I volunteered to do this sometime in the coming week.

**Comment by samreid** [comment]
Here's the last place we published the prototype, see https://github.com/phetsims/circuit-construction-kit-dc/issues/6

**Comment by samreid** [comment]
Proposed version is here: http://www.colorado.edu/physics/phet/dev/html/circuit-construction-kit-dc/1.0.0-dev.157/circuit-construction-kit-dc_en.html

I tested a few minutes in firefox and chrome and didn't see any obvious defects yet.

**Comment by samreid** [comment]
I followed the script in #6 and published dev.157 as the prototype, and it is now available from the PhET website, see http://phet.colorado.edu/sims/html/circuit-construction-kit-dc/latest/circuit-construction-kit-dc_en.html

**Comment by samreid** [comment]
I emailed @kathy-phet and @oliver-phet 

Full details in https://github.com/phetsims/circuit-construction-kit-dc/issues/46

The new prototype (dev.157) replaced the old prototype (dev.5) at the same address:
http://phet.colorado.edu/sims/html/circuit-construction-kit-dc/latest/circuit-construction-kit-dc_en.html

Would you like to notify any users or social media about the change?

Best Regards,
Sam

**Comment by samreid** [comment]
@ariel-phet are you working on this too?

**Comment by ariel-phet** [comment]
@samreid I already made some social media posts.

**Comment by samreid** [comment]
Thanks, closing.
